haskell/private/cc_libraries.bzl

```
@@ -49,7 +49,7 @@ def get_cc_libraries(cc_libraries_info, libraries_to_link):
         if not cc_libraries_info.libraries[cc_library_key(lib_to_link)].is_haskell
     ]
 
-def get_ghci_library_files(hs, cc_libraries_info, libraries_to_link, include_real_paths = False):
+def get_ghci_library_files(hs, cc_libraries_info, libraries_to_link, *, include_real_paths = False, for_th_only=False):
```

```suggestion
def get_ghci_library_files(hs, cc_libraries_info, libraries_to_link, include_real_paths = False, for_th_only=False):
```
Is this needed anywhere?

---

The `*` was there to disallow calls like `get_ghci_library(hs, cc_libs, True)`; I guess I find keyword args more clear when dealing with boolean flags? (and maybe safer when refactoring). Anyway, happy to remove it if you rather

---

Thanks for clarifying, makes sense to enforce this.